Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,69 @@
 ﻿using System;
 
+
 namespace BootCamp.Chapter
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+
+            String firstName = " ";
+            String lastName = " ";
+            int age = 0;
+            float weight = 0.00f;
+            float height = 0.00f;
+
+
+
+            for (int i = 0; i < 2; i++)
+            {
+                Console.Clear();
+                getInput(ref firstName, ref lastName, ref age, ref weight, ref height);
+                String name = firstName + " " + lastName;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'name'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4844457)